plan/main

```diff
@@ -1,24 +1,28 @@
-#include <turnkey/base>
-#include <turnkey/lamp73>
-
-python3-bcrypt
-
-/* install specific versions as using non-standard PHP */
-php7.3-common
-php7.3-curl
-php7.3-json
-php7.3-mbstring
-php7.3-gd
-php7.3-ldap
-php7.3-zip
-php7.3-bcmath
-php7.3-xml
-php7.3-imap
-php7.3-redis
-
-redis-server
-supervisor
-
-unzip
-
-expect      /* Used to interact with interactive php aritsan commands */
+#include <turnkey/base>
+#include <turnkey/mysql>
```

Comment:
FWIW you don't need to include the mysql plan as the lamp81 plan includes it already: https://github.com/turnkeylinux/common/blob/master/plans/turnkey/lamp81#L1